### PR TITLE
add "Create service with template" link

### PR DESCRIPTION
**Pull request by Daxaker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ This guide is intended to be a point of knowledge to everyone who wants to pass
 - Manipulate a running stack of services
 - [Increase number of replicas](https://docs.docker.com/engine/reference/commandline/service_scale/)
 - [Add networks, publish ports](https://docs.docker.com/engine/userguide/networking/)
-- Mount volumes
-- Illustrate running a replicated vs global service
+- [Mount volumes](https://docs.docker.com/engine/admin/volumes/volumes/)
```
Comment (Evalle) [inline README.md]: Please, delete this link

File: README.md [hunk 2]
```diff
@@ -24,13 +24,13 @@ This guide is intended to be a point of knowledge to everyone who wants to pass
...
+- [Illustrate running a replicated vs global service](https://docs.docker.com/engine/swarm/how-swarm-mode-works/services/#replicated-and-global-services)
```
Comment (Evalle) [inline README.md]: And this one

**Comment by Daxaker** [comment]
Done

**Comment by Evalle** [comment]
LGTM, thanks a lot!

**Comment by Daxaker** [comment]
No problem, glad to be helpful

**Pull request merged**